dtreeviz/trees.py

```diff
@@ -214,7 +214,12 @@ def ctree_leaf_distributions(self,
             ax.set_ylabel("samples by class", fontsize=fontsize, fontname=fontname, color=colors['axis_label'])
             ax.grid(visible=grid)
             ax.legend([bar_container0, bar_container1],
-                      [f'class {self.shadow_tree.classes()[0]}', f'class {self.shadow_tree.classes()[1]}'])
+                      [f'class {self.shadow_tree.classes()[0]}', f'class {self.shadow_tree.classes()[1]}'],
```

Comment:
Can we replace this with a call to add_classifier_legend()?

Reply:
Done!

Reply:
![image](https://user-images.githubusercontent.com/4729931/209890973-74c51461-898e-4ca1-bbfe-22f699ad6342.png)